doc/nrf/protocols/matter/overview/bridge.rst

```diff
@@ -0,0 +1,63 @@
+.. _ug_matter_overview_bridge:
+
+Matter Bridge
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter bridge is a device that allows to use non-Matter IoT devices in a Matter fabric together with other native Matter devices.
+The non-Matter devices in such configuration are called bridged devices.
+It is out of the Matter specification scope to define connectivity technologies used by the bridged devices, so they can use any technology (for example Bluetooth LE or Zigbee), as long as bridge device supports it too.
```

Comment:
Maybe it is worth adding here that we operate within the Application Layer only and translate the Matter data model to other Protocol application layers and vice versa, and do not translate protocols themselves within the lower layers.